### DISCUSSION TITLE
Receive alerts via email

### DISCUSSION THREAD

**Discussion by cbenenax2** [simple-login/app#1218] [Q&A]
Is there some way to receive alert notifcations at my mailbox?  I had several alerts that I did not know about until I logged in here!

**Comment by nguyenkims** [comment]
In case of email issues, for example your mailbox is full and bounce all forwarded emails, SimpleLogin can't get in touch with you via emails. Notifications are more reliable in this case. In the future, we plan to send push notification via mobile app as well.
